aliases

```diff
@@ -9,6 +9,7 @@ alias v="$VISUAL"
 alias b="bundle"
 
 # Rails
+alias extension_server="export RUN_SERVER_FOR_CHROME_EXTENSION=true; rake assets:precompile && thin start --ssl -p 7000; export RUN_SERVER_FOR_CHROME_EXTENSION=false;"
```

Comment:
You should be able to achieve this without the `export` command, and without the last `export` resetting to false on the end
`alias extension_server="RUN_SERVER_FOR_CHROME_EXTENSION=true && rake assets:precompile && thin start --ssl -p 7000`